modules/ROOT/pages/composefs.adoc

```diff
@@ -0,0 +1,46 @@
+= ComposeFS
+
+Fedora CoreOS introduced ComposeFS enabled by default starting in Fedora 41. ComposeFS is an overlay filesystem where the data comes from the usual ostree deployement, and
+metadata are in the composeFS file. The result is a truely read-only root (`/`) filesystem, increasing the system integrity and robustness,
+
+This is a first step towards a full verification of filesystem integrity, even at runtime.
+
+== What does it change ? 
+
+The main visible change will be that the root filesystem (/) is now small and full (a few MB, 100% used).
+The real root is mounted in /sysroot and most of the data is stored in /var. 
+
+== How to disable composeFS
+
+ComposeFS can be disabled through a kernel argument: `ostree.prepare-root.composefs=0`.
+
+.Disabling composeFS mount
+[source,yaml,subs="attributes"]
+----
+variant: fcos
+version: {butane-latest-stable-spec}
+kernel_arguments:
+  should_exist:
+    - ostree.prepare-root.composefs=0
+----
+
+== Known issues
+
+=== Kdump
+
+Right now, this prevents kdump from writing the memory dumps as it get confused by the read-only filesystem.
```

Comment:
Isn't the issue during the generation of the initramfs?

Reply:
ah yes, good point. But it's worth noting that setting a remote target (ssh, nfs) is a workaround